src/auth/base-auth-api.ts

```diff
@@ -92,6 +92,7 @@ export class BaseAuthAPI extends BaseAPI {
   clientSecret?: string;
   clientAssertionSigningKey?: string;
   clientAssertionSigningAlg?: string;
+  agent?: unknown;
```

Comment:
Why is this unknown?

Reply:
@gyaneshgouraw-okta Can you explain why is this unknown?

Reply:
@willvedd With the latest set of changes `agent` related new param is not required, I have removed them in this commit - https://github.com/auth0/node-auth0/pull/1002/commits/ffba4af31ec85ea49a6110e07c7e9b8da2dbadba

Initially it was added keeping in mind existing `agent` type as `unknown` [link](https://github.com/auth0/node-auth0/blob/master/src/lib/models.ts#L29)
Ideally it should have been of Agent type derived from [Undici](https://undici.nodejs.org/#/?id=undicifetchinput-init-promise)